views/components/GraphTable/index.jsx

```diff
@@ -1,62 +1,89 @@
 import React from 'react'
-import {
-  TableBody,
-  TableCell,
-  TableHead,
-  TableRow,
-  Table,
-  Paper,
-  Toolbar,
-  Button,
-} from '@mui/material'
-import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
+import { Paper, Button } from '@mui/material'
+import { SaveAlt } from '@mui/icons-material'
+import Table from '../../tables/Table'
+import { SORT_DIRECTION } from '../../../constants'
 
-const GraphTable = ({ graph, onGetCsv }) => {
+import './GraphTable.css'
+
+const GraphTable = ({
+  graph,
+  onGetCsv,
+  page,
+  rowsPerPage,
+  handleChangePage,
+  handleChangeRowsPerPage,
+  comparator,
+  stableSort,
+  sortBy,
+  sortDirection,
+  onSort,
+}) => {
   if (!graph) return null
 
   const { tableColumns, tableRows } = graph.graphTable
+  const headers = [
+    {
+      dataProperty: 'site',
+      label: 'Network',
+      sortable: true,
+    },
+    ...tableColumns.map(({ name }) => {
+      return {
+        dataProperty: name,
+        label: name,
+        sortable: false,
+      }
+    }),
+  ]
+  const visibleRows = React.useMemo(
+    () =>
+      stableSort(tableRows, comparator).slice(
+        page * rowsPerPage,
+        page * rowsPerPage + rowsPerPage
+      ),
+    [sortDirection, sortBy, page, rowsPerPage]
+  )
+  const cellRenderer = (site, property) => {
+    switch (property) {
```

Comment:
why do we need a switch statement here?

Reply:
good point, this is unnecessary 
[3454646](https://github.com/AMP-SCZ/dpdash/pull/695/commits/345464657d6743b4215404773bac8032e82fa604)